addons/payment/models/payment_acquirer.py

```diff
@@ -151,9 +151,9 @@ class PaymentAcquirer(models.Model):
              "Use this field anywhere a small image is required.")
 
     payment_icon_ids = fields.Many2many('payment.icon', string='Supported Payment Icons')
-    payment_flow = fields.Selection(selection=[('s2s','The customer encode his payment details on the website.'),
-        ('form', 'The customer is redirected to the website of the acquirer.')],
-        default='form', required=True, string='Payment flow',
+    payment_flow = fields.Selection(selection=[('form', 'Redirection to the acquirer website (Form mode)'),
```

Comment:
Well Antony asked to remove form mode and server to server mode because it does not mean anything for most people. I agree with him. Form and s2s are technical details you could put into an help. Labels should be understandable by standard people just encoding their paypal (or any other provider) credentials.